native/textern.py

```diff
@@ -64,6 +65,19 @@ def get(self, relfn):
         with open(os.path.join(self.tmpdir, relfn), encoding='utf-8') as f:
             return f.read(), self._tmpfiles[relfn]
 
+    def add_editor(self, absfn, proc):
+        relfn = os.path.basename(absfn)
+        assert relfn in self._tmpfiles
+        self._editors[absfn] = proc
+
+    def kill_editors(self):
+        """Terminate editors that aren't finished yet"""
+        for proc in self._editors.values():
+            proc.terminate()
+        # Try harder
```

Comment:
There should be a timeout between terminate and kill. Or maybe not try harder at all?

Reply:
A timeout, ok. Adding this to my TODO…